src/App.js

```diff
@@ -1,24 +1,31 @@
 import React from 'react';
-import logo from './logo.svg';
+import rtIcon from './tomatillo-icon.png';
 import './App.css';
+import MovieCard from './MovieCard.js';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
+      <header>
+        <section>
+          <h1 class='' id='welcome-msg'>Welcome, User!</h1>
+          <button>Log out, ya dingus</button>
+        </section>
+        <section id='logo'>
+          <img src={rtIcon} alt='The RT Logo'/>
```

Comment:
This was really cool to reverse engineer! I always have such difficulty getting images linked up, and this was surprisingly easy. Glad we got our logo going.